scripts/battlefields/Throne_Room/kindred_spirits.lua

```diff
@@ -0,0 +1,84 @@
+-----------------------------------
+-- Area: Throne Room
+-- BCNM 60 - Kindred Spirits
+-- !pos -111 -6 0.1 165
+-----------------------------------
+local ID = require("scripts/zones/Throne_Room/IDs")
+require("scripts/globals/battlefield")
+require("scripts/globals/items")
+require("scripts/globals/zone")
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId        = xi.zone.THRONE_ROOM,
+    battlefieldId = xi.battlefield.id.KINDRED_SPIRITS,
+    maxPlayers    = 6,
+    levelCap      = 60,
+    timeLimit     = utils.minutes(30),
+    index         = 2,
+    entryNpc      = "_4l1",
+    exitNpcs      = { "_4l2", "_4l3", "_4l4" },
+    requiredItems = { xi.items.MOON_ORB, message = ID.text.A_CRACK_HAS_FORMED },
+
+})
+
+function content:isValidEntry(player, npc)
+    return self.entryNpc == '_4l1'
+end
+
+function content:onEntryEventUpdate(player, csid, option, npc)
+    if Battlefield.onEntryEventUpdate(self, player, csid, option, npc) then
+        if npc:getName() == '_4l1' then
+            self.exitLocation = 1
+            self.lossEventParams  = { [5] = 1 }
+        else
+            self.exitLocation = 0
+            self.lossEventParams  = {}
+        end
+    end
+end
+
+content.groups =
+{
+    { mobs = { "Demons_Elemental" } },
+}
+
+content:addEssentialMobs({ "Count_Andromalius", "Duke_Amduscias", "Grand_Marquis_Chomiel", "Duke_Dantalian" })
```

Comment:
Duke Dantalian can do Astral Flow and I'm not sure if the other mobs can do their job specials.
This means you need to make a mob script for Duke_Dantalian and give him his job special and you need to make a script for "Demons_Avatar" and do something similar.
Look for `mixins = { require("scripts/mixins/job_special") }` and
`mixins = { require("scripts/mixins/families/avatar") }` respectively.

Reply:
When I was doing the testing, they all used their individual 2hr's including spawning a Fenrir and ripping me up. will test again however.

Reply:
You know what that's my bad! I forgot I'm on LSB and didn't have all of the scripts since this BCNM is on ASB. Sorry!